### ISSUE TITLE
Spaces in projects etc.

### ISSUE THREAD

**Issue by orta** [supermarin/xclisten#10]
`xcodebuild -workspace Artsy Folio.xcworkspace -scheme Artsy Folio -sdk iphonesimulator -destination 'name=iPhone Retina (4-inch 64-bit)' test 2> xcodebuild_error.log | xcpretty -tc`

=>

```
xcodebuild: error: Unknown build action 'Folio.xcworkspace'.
```

**Comment by supermarin** [comment]
:+1: will do today

**Comment by supermarin** [comment]
@orta fixed and released in 0.0.7
